controllers/clustertopology_controller_compute_test.go

```diff
@@ -17,15 +17,325 @@ limitations under the License.
 package controllers
 
 import (
+	"context"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
 	. "github.com/onsi/gomega"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
+func TestGetClass(t *testing.T) {
+	fakeScheme := runtime.NewScheme()
+	_ = apiextensionsv1.AddToScheme(fakeScheme)
+	_ = clusterv1.AddToScheme(fakeScheme)
+
+	crds := []client.Object{
```

Comment:
I assume this could be useful for other tests.. what about moving it to a global variable?

Reply:
The only concern I had was the contract labels, which would enforce everyone to handle the ref conversion. But I'll move the CRDs and then adjust a local copy